clients/web/src/models/UserModel.js

```diff
@@ -1,6 +1,26 @@
 girder.models.UserModel = girder.Model.extend({
     resourceName: 'user',
 
+    // Sets this model to the currently logged in user; has no effect is no user
+    // is logged in.
+    current: function (success, error) {
+        girder.restRequest({
+            method: 'GET',
+            path: '/user/me'
+        }).then(_.bind(function (user) {
+            if (user) {
+                this.set(user);
+                if (success) {
+                    success(this);
+                }
+            } else {
+                if (error) {
+                    error(this);
+                }
```

Comment:
Opting for 1 (via 3803da502d3ef8e5179598b87d0f65d0c2f9fa71).  Let me know if the documentation is ok.